branding/src/test/java/com/automationintesting/integration/BrandingServiceIT.returnsBrandingData.approved.txt

```diff
@@ -1,4 +1,5 @@
 {
+    "brandingid": 0,
```

Comment:
So I get that we need a key in the DB, but not sure if sending the brandingid over is necessary. Do you have an intended purpose for this field?

Reply:
Honestly no. This is basically because I copied the RoomDB class unthinkingly.